src/JS/booking.js

```diff
@@ -18,11 +16,13 @@ const renderBookingSalonTitleDate = (salon, title, time) => {
         <h2>salong ${salon}</h2>
         <h1>${title}</h1>
```

Comment:
This function is not really necessary, you could use the "saveBookingInfo" function for getting the data and rendering html but otherwise it's good

Reply:
Sure. I was just thinking that it would be good to have each function just do one thing. I will look into this when I work with the next step to see what will be the best solution.